indico/htdocs/js/indico/jquery/trackrolewidget.js

```diff
@@ -0,0 +1,94 @@
+/* This file is part of Indico.
```

Comment:
Now that you mention it. This should actually go into `htdocs/js/indico/jquery/widgets/`.